### PR TITLE
feat(conf): renamed network configuration properties to make names more intuitive

### PR DESCRIPTION
**Pull request by bluestreak01**
Documentation is already up to date:

https://questdb.io/docs/operations/capacity-planning#network-configuration

Also renamed:

```
http.bind.to -> http.net.bind.to
http.min.bind.to -> http.min.net.bind.to
```

All old property names are also supported. When both properties present in configuration - new names take precedence

To make names consistent with PG Wire config and ILP.

Documentation PR: https://github.com/questdb/questdb.io/pull/693

### CODE REVIEW

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 49 / 50 (98.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/network/IODispatcherConfiguration.java|5|6|83.33%|
|:large_blue_circle:|io/questdb/PropServerConfiguration.java|39|39|100.00%|
|:large_blue_circle:|io/questdb/network/AbstractIODispatcher.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpReceiver.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 60 / 62 (96.77%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/network/IODispatcherConfiguration.java|5|6|83.33%|
|:large_blue_circle:|io/questdb/PropServerConfiguration.java|50|51|98.04%|
|:large_blue_circle:|io/questdb/network/AbstractIODispatcher.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpReceiver.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 54 / 55 (98.18%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/network/IODispatcherConfiguration.java|5|6|83.33%|
|:large_blue_circle:|io/questdb/PropServerConfiguration.java|44|44|100.00%|
|:large_blue_circle:|io/questdb/network/AbstractIODispatcher.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpReceiver.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 54 / 55 (98.18%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/network/IODispatcherConfiguration.java|5|6|83.33%|
|:large_blue_circle:|io/questdb/PropServerConfiguration.java|44|44|100.00%|
|:large_blue_circle:|io/questdb/network/AbstractIODispatcher.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpReceiver.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 54 / 55 (98.18%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/network/IODispatcherConfiguration.java|5|6|83.33%|
|:large_blue_circle:|io/questdb/PropServerConfiguration.java|44|44|100.00%|
|:large_blue_circle:|io/questdb/network/AbstractIODispatcher.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpReceiver.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

coverage check fail. please retry. :fearful:

[Please let me know](https://github.com/naver/cover-checker/issues/new) when error again.

com.naver.nid.cover.parser.coverage.exception.ParseException: java.io.FileNotFoundException: /home/vsts/work/1/s/core/CCReport43F6D5EF/jacoco.xml (No such file or directory)

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 54 / 55 (98.18%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/network/IODispatcherConfiguration.java|5|6|83.33%|
|:large_blue_circle:|io/questdb/PropServerConfiguration.java|44|44|100.00%|
|:large_blue_circle:|io/questdb/network/AbstractIODispatcher.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpReceiver.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 54 / 55 (98.18%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/network/IODispatcherConfiguration.java|5|6|83.33%|
|:large_blue_circle:|io/questdb/PropServerConfiguration.java|44|44|100.00%|
|:large_blue_circle:|io/questdb/network/AbstractIODispatcher.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpReceiver.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 54 / 55 (98.18%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/network/IODispatcherConfiguration.java|5|6|83.33%|
|:large_blue_circle:|io/questdb/PropServerConfiguration.java|44|44|100.00%|
|:large_blue_circle:|io/questdb/network/AbstractIODispatcher.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpReceiver.java|1|1|100.00%|

File: core/src/main/resources/io/questdb/site/conf/server.conf [hunk 1]
```diff
@@ -13,7 +13,31 @@
 #http.enabled=true
 
 # IP address and port of HTTP server
-#http.bind.to=0.0.0.0:9000
+#http.net.bind.to=0.0.0.0:9000
```
Comment (ideoma) [inline core/src/main/resources/io/questdb/site/conf/server.conf:16]: This has high chances to be changed from default in some installations already. Can we make it backward compatible or avoid changing?

Comment (bluestreak01) [reply core/src/main/resources/io/questdb/site/conf/server.conf:16]: > All old property names are also supported.

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 54 / 55 (98.18%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/network/IODispatcherConfiguration.java|5|6|83.33%|
|:large_blue_circle:|io/questdb/PropServerConfiguration.java|44|44|100.00%|
|:large_blue_circle:|io/questdb/network/AbstractIODispatcher.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpReceiver.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 54 / 55 (98.18%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/network/IODispatcherConfiguration.java|5|6|83.33%|
|:large_blue_circle:|io/questdb/PropServerConfiguration.java|44|44|100.00%|
|:large_blue_circle:|io/questdb/network/AbstractIODispatcher.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpReceiver.java|1|1|100.00%|

**Review by glasstiger — APPROVED** [review]

**Pull request merged**